kythe/cxx/indexer/cxx/IndexerASTHooks.h

```diff
@@ -683,8 +685,7 @@ class IndexerASTVisitor : public RecursiveTypeVisitor<IndexerASTVisitor> {
             std::unique_ptr<IndexerWorklist> NewWorklist) {
     Worklist = std::move(NewWorklist);
     Worklist->EnqueueJob(std::make_unique<IndexJob>(InitialDecl));
-    while (!options_.ShouldStopIndexing() && Worklist->DoWork())
-      ;
+    while (!options_.ShouldStopIndexing() && Worklist->DoWork());
```

Comment:
make this a `{}` if clang-format wants to hide it: https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching